vector/src/test/java/im/vector/app/core/utils/CheckWebViewPermissionsUseCaseTest.kt

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (c) 2022 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package im.vector.app.core.utils
+
+import android.app.Activity
+import android.content.pm.PackageManager
+import android.net.Uri
+import android.webkit.PermissionRequest
+import androidx.core.content.ContextCompat.checkSelfPermission
+import io.mockk.MockKAnnotations
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.mockkStatic
+import io.mockk.unmockkStatic
+import org.amshove.kluent.shouldBe
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+
+class CheckWebViewPermissionsUseCaseTest {
+
+    private val checkWebViewPermissionsUseCase = CheckWebViewPermissionsUseCase()
+
+    private val activity = mockk<Activity>().apply {
+        every { applicationContext } returns mockk()
+    }
+
+    @Before
+    fun setup() {
+        MockKAnnotations.init(this)
```

Comment:
I am not sure this is needed. `MockKAnnotations.init()` is useful when we use the annotations for mockking objects like `@MockK`.

Reply:
Yep, I started with using `@Mockk` then forgot to remove it. Done.